server.py

```diff
@@ -104,18 +105,18 @@ def do_OPTIONS(self):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('ip', help='switcher IP address')
-    parser.add_argument('passphrase', help='passphrase to compare requests to', type=str, default='Password1')
+    parser.add_argument('--ip', help='switcher IP address', default='127.0.0.1')
```

Comment:
What's the thought process behind using localhost as the default here?

Reply:
Not really any? I had it as a default argument in the original `entrypoint.sh` I created when I first authored the Dockerfile:

```bash
SERVER_IP="${SERVER_IP:-${1:-127.0.0.1}}"
```

Could make that a required command line argument as one possibility but generally you just pass arguments into containers as environment variables rather than changing the cmd arguments if possible. My specific deployment of pyatem I'm using is in my home on a kubernetes cluster so the environment variables are defined in a kubernetes secret and mounted into the pod/container as environment variables.

Reply:
Got it. I'm going to omit that since the only time the ATEM would be at local host would be when running a simulator (and the simulator isn't too reliable).